.circleci/config.yml

```diff
@@ -83,7 +83,7 @@ jobs:
 
       - run:
           name: Save pkg version to a file
-          command: node ./.circleci/scripts/read-changelog.js > .pkg-version
+          command: node -pe "require('./package.json').version" > .pkg-version
```

Comment:
In [neticrm repo](https://github.com/twreporter/neticrm/blob/d588bfe1ce1410a813f397cc5a63b77e06f25bc1/.circleci/config.yml#L12), I use `npm show . version` to get version, I think we could discuss a single way and apply it all. 
How do u think about it ?

Reply:
`npm show . version` is also a good way but the package needs to be published to npm registry, for example, `plate` repo is not published and `npm show . version` gets error.

Reply:
btw, I tried `npm show . version` in netiCRM repo & the version is actually from this [library](https://www.npmjs.com/package/sync-tool), which is irrelevant to our repo